src/Anonymization/Anonymizer/Core/Md5Anonymizer.php

```diff
@@ -21,10 +25,21 @@ class Md5Anonymizer extends AbstractAnonymizer
     public function anonymize(Update $update): void
     {
         $expr = $update->expression();
+        $columnExpr = $expr->column($this->columnName, $this->tableName);
 
-        $update->set(
-            $this->columnName,
-            $expr->md5($expr->column($this->columnName, $this->tableName)),
-        );
+        if ($this->options->get('use_salt', true)) {
+            $columnExpr = $expr->concat($columnExpr, $expr->value($this->getSalt()));
+
+            $update->set(
+                $this->columnName,
+                // Work around some RDBMS not seeing the NULL value anymore
+                // once we added the string concat.
+                $this->getSetIfNotNullExpression(
+                    $expr->md5($columnExpr)
+                ),
```

Comment:
why is this needed here and not on the EmailAnonymizer ? :thinking:

Reply:
It is already in email anonymizer if I remember correctly.

Reply:
Yes, I can confirm that.